corehq/apps/sms/views.py

```diff
@@ -1915,6 +1918,30 @@ def dispatch(self, request, *args, **kwargs):
         return super(SMSSettingsView, self).dispatch(request, *args, **kwargs)
 
 
+@method_decorator(toggles.ICDS_CUSTOM_SMS_REPORT.required_decorator(), name='dispatch')
+class SMSUsageReport(BaseMessagingSectionView):
```

Comment:
Similar to my comment on the template, I think this belongs in `custom/icds`, not in `corehq`.

Reply:
Done